### ISSUE TITLE
Implement Ubuntu VirtualBox VM on OSX

### ISSUE THREAD

**Issue by jeff1evesque** [jeff1evesque/LeQue#18]
We will move our local environment from a MAMP instance on the OSX (host) to an Ubuntu VirtualBox VM.  This way, we can install _PocketSphinx_ (https://github.com/jeff1evesque/audio-analyzer/issues/17).

**Comment by jeff1evesque** [comment]
We've downloaded _VirtualBox 4.3.12_ `64-bit` (https://www.virtualbox.org/wiki/Downloads).  Next, we download _Ubuntu_ iso file (http://www.ubuntu.com/download/desktop/contribute/?version=14.04&architecture=amd64)

Once _Ubuntu's iso_ file has been downloaded, open VirtualBox. Click on the new button in the top left corner.

As we configure a new Virtual Machine, select all the default settings.  Unselect boot from _Floppy_, and move _Hard Disk_ option as the first boot order:

![screen shot 2014-05-24 at 12 42 35 pm](https://cloud.githubusercontent.com/assets/2907085/3075080/992df9a0-e362-11e3-8fda-e39a7d5d6cd6.png)

Finally, add a new device to the _Controller: IDE_, and have it point to where we saved the newly installed _Ubuntu 14.04_ iso file:

![screen shot 2014-05-24 at 12 41 50 pm](https://cloud.githubusercontent.com/assets/2907085/3075091/7eeafee8-e363-11e3-802e-b5111a4ea244.png)

![screen shot 2014-05-24 at 12 55 05 pm](https://cloud.githubusercontent.com/assets/2907085/3075104/484f2c46-e364-11e3-8553-542320c062b7.png)

![screen shot 2014-05-22 at 5 42 33 pm](https://cloud.githubusercontent.com/assets/2907085/3060766/5de22ff0-e1fb-11e3-8302-455fb72e619f.png)

Once VirtualBox boots up, choose to _Install Ubuntu_, rather than _Try Ubuntu without installing_.
- http://www.wikihow.com/Install-Ubuntu-on-VirtualBox

**Comment by jeff1evesque** [comment]
Since we couldn't get _Unity 3D_ working, we will remove the _Ubuntu Guest_ profile we created above by right clicking the profile and select _Remove..._, then _Delete all files_ :

![screen shot 2014-05-27 at 3 16 01 pm](https://cloud.githubusercontent.com/assets/2907085/3096388/67027104-e5d3-11e3-811d-63ff43411044.png)

**Comment by jeff1evesque** [comment]
Now, we will follow the same steps outlined above https://github.com/jeff1evesque/audio-analyzer/issues/18#issuecomment-43914218, except when we add a new device to the _Controller: IDE_ we will point it to the file `ubuntu-13.04-desktop-amd64+mac.iso` instead of `ubuntu-14.04-desktop-amd64.iso`.  The former file can be downloaded from http://releases.ubuntu.com/13.04/, by selecting _64-bit Mac (AMD64) desktop image_.

---

We chose _Ubuntu 13.04_ this time around, because https://github.com/jeff1evesque/audio-analyzer/issues/23#issuecomment-44152422, specifically the instruction from:

http://namhuy.net/951/how-to-fix-slow-performance-ubuntu-13-04-running-in-virtualbox.html 

implies this version of Ubuntu.

**Comment by jeff1evesque** [comment]
We downgraded and now have a working environment where the CPU is much less than 30% (when OS is idle):
- from VirtualBox 4.3.12 -> 4.1.10 r76795
- from Ubuntu 14.04 -> 10.04

![screen shot 2014-05-30 at 10 28 05 pm](https://cloud.githubusercontent.com/assets/2907085/3138054/3aa770f6-e86b-11e3-8837-505e20f89865.png)

---

**Note 1:** we attempted unsuccessfully Ubuntu version 14.04 and 13.04 (VirtualBox 4.3.12), 11.04 (VirtualBox 4.1.10 r76795 - didn't accept the _iso_ file for Ubuntu 11.04) before successfully implementing version 10.04.  We also did not attempt to enable _Unity 3D_ (https://github.com/jeff1evesque/audio-analyzer/issues/23).

**Note 2:** when a new process starts, it can take up to 80-100% of the CPU.  But, this does not last more than a second or two, then it calms down to about 17.5%.  Keep in mind, we are running a very old OSX host -- specifically, 10.6.8.
